docs/plat/rpi3.rst

```diff
@@ -101,7 +101,7 @@ secure platform!
 
     0x00000000 +-----------------+
                |       ROM       | BL1
-    0x00010000 +-----------------+
+    0x00020000 +-----------------+
```

Comment:
You will need to update the documentation, the "truncate" command now needs to truncate to 128K.

Reply:
Actually, it would also be nice if you could add an example to the documentation about how to compile the TF with TBB for this platform.